core/server/master/src/main/java/alluxio/master/metastore/rocks/RocksBlockMetaStore.java

```diff
@@ -89,11 +95,14 @@ public RocksBlockMetaStore(String baseDir) {
     RocksDB.loadLibrary();
     // the rocksDB objects must be initialized after RocksDB.loadLibrary() is called
     mDisableWAL = new WriteOptions().setDisableWAL(true);
+    mToClose.add(mDisableWAL);
```

Comment:
now i'm adding all RocksObjects into this list instead of closing them one by one in `close()`

Reply:
Actually i prefer this style where we create something and immediately put it into the record. Imagine my brain only has a register of size one.  And I will panic if i don't deal with this obj immediately because i know I for sure will forget.